docs/administrator-guide/notification-management.md

```diff
@@ -54,6 +54,20 @@ If you’d like to see support for additional events, [let us know](https://matt
 
 ![This is the Channel Subscription modal](../.gitbook/assets/channel-subscriptions-modal.png)
 
+## How to create and use the subscription modal?
+
+1. Type the `/jira subscribe` command to open the "Create subscription" modal in the particular channel.
+2. Click on the **Create Subscription** button to create the subscription and receive the Jira issue notifications.
+3. Write the name of the subscription in the **Subscription Name** field.
+4. Select the project name to which you want to subscribe.
+5. Select the events and issue types for which you want to receive the notifications in the Mattermost channel.
+6. To be more specific, you can add filters as well. You can either include or exclude the particular filter by adding its type or name. For now, we have 3 types of filters :
+   * Epic Link
+   * Labels
+   * Priority
```

Comment:
> For now, we have 3 types of filters :

There many other filter fields available, including custom fields. As mentioned in https://github.com/mattermost/mattermost-plugin-jira/pull/583, "Basically all of them except date pickers." I know that Status is also not currently supported.

What fields have you tried to use that are not available in the feature?

Reply:
@mickmister 
After exploring now I am getting total 10 types of filters:
1. Affects versions
2. Epic Link
3. Fix versions
4. Labels
5. Priority

And a few custom fields as well.
1. Checkboxes
2. Labels
3. Radio Buttons
4. Select List (multiple choices)
5. Select List (single choice)